src/components/homepage/HomePage.jsx

```diff
@@ -13,7 +17,7 @@ export default function HomePage() {
                 <div className="grand-hero-content">
                     <h1>A Video Library for all Chess Lover</h1>
                     <div className="p-2"></div>
-                    <button class="btn btn-primary hero-button">Explore</button>
+                    <button class="btn btn-primary hero-button" onClick={()=>navigate("/explore")}>Explore</button>
```

Comment:
I don't think here you need to use useNavigate(), because it is should be link or Navlink from react-router-dom